contrib/completion/bash/docker

```diff
@@ -1396,7 +1396,7 @@ _docker_network_ls() {
 
 	case "$prev" in
 		--filter|-f)
-			COMPREPLY=( $( compgen -S = -W "id name type" -- "$cur" ) )
+			COMPREPLY=( $( compgen -S = -W "driver id label name type" -- "$cur" ) )
```

Comment:
@thaJeztah Hey, that's cool. Your're diving into the completions.
The completion is correct.
You could also add the secondary completion to https://github.com/docker/docker/blob/master/contrib/completion/bash/docker#L1380-1395, see https://github.com/docker/docker/blob/master/contrib/completion/bash/docker#L1327 for how to get the possible values.
Or just give me a ping if I should add this later.